spec/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior_spec.rb

```diff
@@ -5,46 +5,75 @@
 RSpec.describe Wings::PcdmValkyrieBehavior do
   subject(:factory) { Wings::ModelTransformer.new(pcdm_object: pcdm_object) }
 
-  context 'for PCDM & Hydra-Works models', :clean_repo do
```

Comment:
I got rid of the context which would apply to all tests in this file.  It was a carry over from starting these tests in model_transformer_spec.  This cause the indentation to change which makes it look like more changed than it did.  Starting here, this is only an indentation change.